a2-synchrotron-contiki/core/net/mac/chaos/chaos.h

```diff
@@ -220,8 +221,9 @@ enum {
 #define INITIATOR_NODE_ID ((uint16_t)INITIATOR_NODE)
 #endif
 
-//#define IS_INITIATOR()              (node_id == INITIATOR_NODE_ID)
-#define IS_INITIATOR()              (node_id == 3 || node_id == 4)
+#ifndef CHAOS_CLUSTER
```

Comment:
Look into how to define INITIATOR here even if CHAOS_CLUSTER is defined, it is ugly to define it in two separate files